mod/developers/actions/developers/entity_explorer_delete.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+$guid = (int) get_input('guid');
+$type = get_input('type');
+$key = get_input('key');
+
+$show_hidden = access_show_hidden_entities(true);
+
+$entity = get_entity($guid);
+if (empty($entity) || empty($type) || $key === null) {
+	register_error(elgg_echo('error:missing_data'));
+	access_show_hidden_entities($show_hidden);
+	forward(REFERER);
```

Comment:
We should start using response objects for new actions.
`return elgg_error_response()`